source/macos/AdaptiveCards/AdaptiveCards/Renderers/ColumnSetRenderer.swift

```diff
@@ -30,17 +30,21 @@ class ColumnSetRenderer: BaseCardElementRendererProtocol {
             // Check if has extra properties else add column view
             guard index > 0, column.getSpacing() != .none, !column.getSeparator() else {
                 columnSetView.addArrangedSubview(columnView)
+                BaseCardElementRenderer.shared.configBleed(collectionView: columnView, parentView: columnSetView, with: hostConfig, element: column, columnView: nil)
                 continue
             }
             let wrappingView = ACRContentStackView(style: column.getStyle(), hostConfig: hostConfig)
             wrappingView.translatesAutoresizingMaskIntoConstraints = false
             wrappingView.orientation = .horizontal
             wrappingView.addSpacing(column.getSpacing())
             wrappingView.addSeperator(column.getSeparator())
+            wrappingView.wantsLayer = true
+            wrappingView.layer?.backgroundColor = columnView.layer?.backgroundColor
             
             wrappingView.addArrangedSubview(columnView)
             columnView.trailingAnchor.constraint(equalTo: wrappingView.trailingAnchor).isActive = true
             columnSetView.addArrangedSubview(wrappingView)
+            BaseCardElementRenderer.shared.configBleed(collectionView: wrappingView, parentView: columnSetView, with: hostConfig, element: column, columnView: columnView)
```

Comment:
```suggestion
            columnSetView.addArrangedSubview(wrappingView)
            BaseCardElementRenderer.shared.configBleed(collectionView: columnView, parentView: columnSetView, with: hostConfig, element: column, columnView: nil)
```
What happens if we just pass `columnView` instead of `wrappingView`?